doc/themes/the-things-stack/layouts/partials/head.html

```diff
@@ -30,4 +30,5 @@
   <meta name="twitter:title" content="{{ .Title }}" />
   <meta name="twitter:description" content="{{ $scratch.Get "description" }}" />
   <meta name="twitter:card" content="summary" />
+  {{ template "_internal/google_analytics_async.html" . }}
```

Comment:
Enabling Google Analytics means that we have to ask for cookie consent if we want to be GDPR-compliant.

Reply:
removing the template for now. I'll leave the config field and we can add the environment variable in CI so when @wienke decides how to do GDPR compliance it's easy to add back in